### PR TITLE
[Intel MKL] Fixing Windows build with MKL-DNN support

### PR DESCRIPTION
**Pull request by mahmoud-abuzaina**
This PR is making few changes to code that MSVC compiler did not like.

### CODE REVIEW

File: tensorflow/core/kernels/mkl_conv_ops.cc [hunk 1]
```diff
@@ -1571,9 +1565,11 @@ class MklQuantizedConv2DOp
         output_max->flat<float>()(0) = max_output_value;
       } else {
         size_t depth = min_filter.NumElements();
-        AllocateOutputSetMklShape(context, 1, &output_min, {depth},
+        AllocateOutputSetMklShape(context, 1, &output_min,
+                                  {static_cast<ptrdiff_t>(depth)},
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_conv_ops.cc]: I'm curious. Why does this have to be `ptrdiff_t`?

**Review by penpornk — COMMENTED** [review]
Thank you for your PR! Could you please paste what MSVC complains here, for educational purpose?

File: tensorflow/core/kernels/mkl_conv_ops.cc (hunk 1 above)
Comment (mahmoud-abuzaina) [reply tensorflow/core/kernels/mkl_conv_ops.cc]: error C2398: Element '1': conversion from 'std::size_t' to 'std::ptrdiff_t' requires a narrowing conversion

Comment (penpornk) [reply tensorflow/core/kernels/mkl_conv_ops.cc]: Thank you!

**Review by penpornk — APPROVED** [review]

**Pull request merged**